app/assets/javascripts/shared/datatable/state.js

```diff
@@ -72,6 +72,7 @@ DradisDatatable.prototype.updateRecordState = function (newState) {
       data: {
         ids: DradisDatatable.prototype.rowIds(selectedRows),
         state: newState,
+        return_to: $('[data-behavior=qa-viewer]').length > 0 ? 'qa' : null
```

Comment:
Should this be `~=` to allow for a case where we add additional data-behavior attributes to the qa-viewer container in the future?